benchmark/_dummy-benchmarker.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
not intending to be pedantic... but can we use `_noop-benchmarker` (noop for non-operational) instead of `_dummy-...`.